source/globus/cli.rst

```diff
@@ -1,94 +1,95 @@
 .. _cli:
 
+============================
 Command Line Interface (CLI)
-----------------------------
+============================
 
-In addition to the web interface, also a set of Globus CLI tools are available.
-These are useful for automating your transfer processes, including the
-scheduling of recurrent transfers.
+The Globus CLI allows you to manage and transfer data from the command line.
+It is available for Windows, Mac and Linux (examples below taken from Linux).
 
 
 Getting started with the CLI
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The `Globus CLI documentation`_. contains an adequate introduction to the CLI.
-The following only includes VSC-specific examples and tips.
+----------------------------
 
-The latest release can be locally installed in e.g. a Python Virtual
-Environment as follows::
+The Globus CLI can be installed as a python package.
```

Comment:
could you also install it as a module and add it to the docs?

there are already easyconfigs for it in the EB repo, and it's already installed in Hydra:
```
module load Globus-CLI
```

Reply:
Thanks for the suggestion, I wasn't aware!

We will merge the documentation for now (we need it online today for a user) and install the module either Friday or beginning or January (depending on some other tasks).

Reply:
Mentioned the CLI module on Hydra now

Reply:
thanks!